update contact section of the html  the css styles of the contact se…

ANTOINE1128 opened the pull request:
add  contact section in the HTML file
add style to the HTML contact section

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
+                        <h3 class ="contact-description">
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
```
- [ ] Your form section should be in a `form` element, this is good for semantic and effectivity reasons, check [this article](https://social.msdn.microsoft.com/Forums/en-US/500e1609-612d-41a5-bb5b-30b83ee40b90/form-and-div-differences?forum=aspwebforms) for more.

- It (the form) should likewise have an `action` attribute leading to a FormSpree submission, and a `method` attribute, with the value `POST` to indicate that data is been submitted to the database.

Check out [how to use form spree here!](https://www.youtube.com/watch?v=gWgzzVVcqfA)

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
+                            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                        </h3>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
```
- [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

AdnanOlarmmi replied:
- [ ] Kindly research and add the limitation for maximum length, as it is a requirement 😉.

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
+                        <ul class="input-filed">
+                        <li><label for="email"> <input type="email" id="email" name="email" placeholder="Full name"></label></li>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
```
- [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
+                       <li> <label for="e-mail">
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
+                       <textarea name="comment" id="comment" cols="30" rows="10">write me something</textarea>
```
- [ ] As it is in the requirement, all the input fields should have the `required` attribute, to ensure that proper information or data has been retrieved.

AdnanOlarmmi replied:
- [ ] The `textarea` element should also be in between the `form` element. 😉

AdnanOlarmmi replied:
- [ ] Kindly research and add the limitation for maximum length, as it is a requirement 😉.

AdnanOlarmmi commented on index.html:
- As per the design. 

- What you submitted
![image](https://user-images.githubusercontent.com/41163571/229572326-b555170d-2046-4106-9219-a94876fd13f4.png)

- What is expected
<img width="209" alt="right" src="https://user-images.githubusercontent.com/41163571/228357941-50bd4f8a-5229-4bd8-90f3-83362a3de0ec.png">

- Kindly fix all aspect of the design to match the Figma design as much as possible, 🏃

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,31 @@ <h1 class="title">About me</h1>
+                        <input type="text" name="e-mail" id="e-mail" placeholder="e-mail address"></label></li>  
+                     </ul>
+                       <textarea name="comment" id="comment" cols="30" rows="10">write me something</textarea>
+                       <button class="getintouch">get in touch<button>
```
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed`, and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/229573539-fc09fa49-7de0-4b4c-b6c7-b497ec95e922.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">

- And please remove the black border around the button, and fix the capitalization of the text in the button.

AdnanOlarmmi reviewed (changes requested):
Hi @ANTOINE1128, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

![You're Almost there, funny one eyed alien](https://user-images.githubusercontent.com/41163571/217957948-6d8d9f33-125c-4724-954f-330eed60533c.gif)

## Highlights
- Keep Improving.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] I suggest that you take Google or other available search engines as your friend in your developer journey, as a developer *you aren't paid for what you know, but where you can find the answer*, please never give up. Most developers don't know this from the top of their head, but by researching, asking questions and making your finding, things will become more understandable. Please take this advice of mine as a little token from me to you. 😃 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ANTOINE1128 commented:
- added the form section 
- added  an action attribute leading to a FormSpree submission, and a method attribute, with the value POST to indicate that data is been submitted to the database
- textarea element is also in between the form element.
- As it is in the requirement, all the input fields have the required attribute
- add the limitation for maximum length
-  fixed all aspect of the design to match the Figma design as much as possible
-  removed the black border around the button, and fix the capitalization of the text in the button.
- the buttons also follow the design guidelines, such as enabled, hover, pressed, and disabled

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
+                        <br>
+                        <label for="getintouch"></label>
+                        <input type="submit"  value="Get in touch" id="getintouch">
```
- [ ] Please kindly ensure that the button matches the given Interactions on Figma.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/229857403-392b3e18-5928-4903-8850-1a9d24fa97da.gif)

- Figma design
<img width="274" alt="button2" src="https://user-images.githubusercontent.com/41163571/227492511-ac99583d-add1-41a8-b99f-2b2f12771941.png">

- And it seems you are confusing clicked to disable 😃

AdnanOlarmmi commented on index.html:
```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
+
+                        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
```
- [ ] Kindly remove the little black border around the `textarea` to perfectly match the design.

<img width="149" alt="image" src="https://user-images.githubusercontent.com/41163571/229858270-89a67cf6-f845-43f3-aebe-a5a4f3d1cb9d.png">

AdnanOlarmmi replied:
- [ ] Kindly add the left margins to the input fields to match the design.

- Your design
<img width="302" alt="image" src="https://user-images.githubusercontent.com/41163571/229858505-ca00765d-c5c6-460d-9b46-e9422333cd36.png">

- Figma design
![image](https://user-images.githubusercontent.com/41163571/229572326-b555170d-2046-4106-9219-a94876fd13f4.png)

AdnanOlarmmi reviewed (changes requested):
Hi @ANTOINE1128, 

Hello there again 😃
This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

![You're Almost there, funny one eyed alien](https://user-images.githubusercontent.com/41163571/217957948-6d8d9f33-125c-4724-954f-330eed60533c.gif)

## Highlights
- Nice one using FormSpree.
- Nice design so far.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

V-Blaze reviewed (approved):
Hi @ANTOINE1128, @gisachris  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![giphy](https://user-images.githubusercontent.com/107632780/229644741-7dea3027-bf3f-4416-855b-46095e3bf6d5.gif)

## Highlights

- Implemented changes requested by the previous reviewer💯
- Ui matches the Figma design 👍
- Nice and clean code 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
